boards/silabs/radio_boards/siwx917_rb4338a/siwx917_rb4338a.dts

```diff
@@ -67,4 +82,9 @@
 			pinmux = <ULPUART_RX_ULP9>;
 		};
 	};
+	ulpi2c_default: ulpi2c_default {
+		group {
+			pinmux = <ULPI2C_SDA_ULP6>, <ULPI2C_SCL_ULP7>;
```

Comment:
It comes down to whether we'll enable any of the flags or not in the future. If we make use of flags, subgroups are necessary to give different flags to different pins. It is possible to configure pullups, drive strength, slew rate and a few other pin properties using flags, several of which are possible to implement on 917. Since the group-based format is the one most pinctrl drivers use, I prefer keeping it in order to be forward compatible.